src/views/virtualmachines/list/hooks/useSelectedFilters.ts

```diff
@@ -0,0 +1,31 @@
+import { useMemo } from 'react';
+import { useSearchParams } from 'react-router-dom-v5-compat';
+
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { RowFilter } from '@openshift-console/dynamic-plugin-sdk';
+
+const useSelectedFilters = (
+  rowFilters: RowFilter<V1VirtualMachine>[],
+  searchFilters: RowFilter<V1VirtualMachine>[],
+) => {
+  const [searchParams] = useSearchParams();
+
+  return useMemo(() => {
+    const selectedFilters = rowFilters.map((filter) =>
+      searchParams.get(`rowFilter-${filter.type}`) ? filter.type : null,
+    );
+
+    selectedFilters.push(
+      ...(searchFilters || []).map((filter) =>
+        searchParams.get(filter.type) ? filter.type : null,
+      ),
+    );
+
+    searchParams.get('name') && selectedFilters.push('name');
+    searchParams.get('labels') && selectedFilters.push('labels');
```

Comment:
Can you please add consts for name and labels?

Reply:
+1

Reply:
fixed